app/js/lib/foxbox/services/base.js

```diff
@@ -112,34 +114,72 @@ export default class BaseService extends EventDispatcher {
   /**
    * Setups value watcher for the getter matching specified selector.
    *
-   * @param {Object} selector Selector to match getter which value we would like
-   * to watch.
+   * @param {string} alias Watcher alias to match getter which value we would
+   * like to watch.
    * @param {function} handler Function to be called once getter value changes.
    */
-  watch(selector, handler) {
+  watch(alias, handler) {
```

Comment:
That my attempt to have the following behavior:
* Automatically start watching only when consumer wants to watch (e.g. when view does ```motionSensorService.watch('motion', ....)```, so service constructor will no longer forcibly start watching.
* Automatically stop watching when consumer doesn't want to watch anymore (e.g. when view does ```motionSensorService.unwatch('motion', .....)```;
* When service is being destroyed we can remove all event listeners in one place and log warning about forgotten event listener.

To do this my idea is to:
* Define which watchers service provides, what alias can be used by consumer (view) and how raw getter value is processed before it's returned to the consumer. Like this:
```
super(props, api, undefined, new Map([
  ['motion', ['OpenClosed', p.onMotionStateChanged]],
]));
```

* So consumer just does ```motionSensorService.watch('motion', ....)``` and receives boolean instead of ```{ OpenClosed: "Open"}```

Definitely ```watch\unwatch``` is much less flexible comparing to ```on\off\once\offAll```, but I chose a tradeoff solution (if that watcher subsystem proves to be good, we can switch back to ```on/offf``` and friends, it just requires more effort). Let me know your thoughts!